frontend/src/Metamaps/Listeners.js

```diff
@@ -19,7 +19,7 @@ const Listeners = {
       if (!(Active.Map || Active.Topic)) return
 
       const onCanvas = e.target.tagName === 'BODY'
-
+      
```

Comment:
If you run `node_modules/.bin/eslint --fix frontend/src/Metamaps/Listeners.js` it'll automatically remove these trailing spaces. You can also just run `node_modules/.bin/eslint --fix frontend` to check the whole folder. The equivalen for rubocop is `rubocop -a` or `rubocop -a path/to/file.rb`